### PR TITLE
Update licenses.json to 3.5-1-g1266c6b

### PR DESCRIPTION
**Pull request by carmenbianca**

### CODE REVIEW

**Comment by carmenbianca** [comment]
I think this PR adequately fixes the outdatedness of the json file and the absence of EXCEPTION_MAP. I need the up-to-date versions of LICENSE_MAP and EXCEPTION_MAP in a project I'm (re-)releasing soon.

**Comment by pombredanne** [comment]
@carmenbianca thank you ++

File: tests/test_document.py [hunk 1]
```diff
@@ -276,5 +276,14 @@ def test_from_identifier(self):
         assert mit.url == 'http://spdx.org/licenses/MIT'
 
 
+class TestException(TestCase):
+
+    def test_exception_list(self):
+        assert EXCEPTION_MAP["Linux Syscall Note"] == "Linux-syscall-note"
```
Comment (pombredanne) [inline tests/test_document.py]: nit pikcing... do you mind using single quotes for strings to stay consistent with the rest of the code?

Comment (carmenbianca) [reply tests/test_document.py]: Pushed 0125f07

**Comment by pombredanne** [comment]
Thank you ++

**Pull request merged**